src/main/java/org/folio/circulation/domain/notice/NoticeContextUtil.java

```diff
@@ -96,7 +99,11 @@ private static JsonObject createItemContext(Item item) {
       .put("loanType", item.getLoanTypeName())
       .put("copy", captionsToken)
       .put("numberOfPieces", item.getNumberOfPieces())
-      .put("descriptionOfPieces", item.getDescriptionOfPieces());
+      .put("descriptionOfPieces", item.getDescriptionOfPieces())
+      .put("effectiveLocationSpecific", location.getName())
```

Comment:
What happens if the location is null?

Reply:
NullPointerException will be thrown. Pushed the changes, now location is optional.